app/grandchallenge/charts/static/js/charts/render_charts.mjs

```diff
@@ -4,9 +4,15 @@ function renderVegaLiteChart(element) {
 }
 
 document.addEventListener("DOMContentLoaded", function(event) {
-    for (const element of document.getElementsByClassName("vega-lite-chart")) {
-        renderVegaLiteChart(element);
+
+    var tabsElement = document.getElementById("v-pills-tab");
+
+    if(tabsElement == null) {
```

Comment:
Using >ES6 javascript this would become:

```suggestion
    const tabsElement = document.getElementById("v-pills-tab");

    if(tabsElement === null) {
```

Or just simplify it to:

```suggestion
    if(document.getElementById("v-pills-tab") === null) {
```

Reply:
Done, I did it like you suggested